src/client/app/core/core.module.ts

```diff
@@ -32,8 +32,9 @@ import {
   MdSlideToggleModule,
   MdToolbarModule
 } from '@angular/material';
-import { CoreRoutingModule } from './core-routing.module';
+import { CoreRoutingModule, routingComponents } from './core-routing.module';
```

Comment:
Hier willst Du "routingComponents" importieren, die exportier "core-routing.module" aber nicht!
Also ist das Kunst oder kann das weg?

Reply:
Lieber @rpriber 
Letzteres ;-) . Ich habe gleich noch ein wenig aufgeräumt und dabei bemerkt, dass `core-routing.module.ts` eine Menge Komponenten exportiert, die in `core.module.ts` nicht benötigt werden. Aber offensichtlich habe ich das Import-Äquivalent in `core.module.ts` vergessen zu entfernen... Vielen Dank für den Hinweis, ich werds gleich nochmals hochladen!